### PR TITLE
Add protocol guild members set

### PR DESCRIPTION
**Pull request by DanTehrani**

### CODE REVIEW

File: packages/frontend/src/lib/sets.ts [hunk 1]
```diff
@@ -148,6 +152,12 @@ export const SET_METADATA: { [key: string]: SetMetadata } = {
     description: 'Was one of the first 500 beacon chain genesis stakers',
     displayName: 'First 500 Beacon Chain Genesis Staker',
   },
+  'protocol-guild': {
+    count: 152,
+    duneURL: 'https://app.splits.org/accounts/0x84af3D5824F0390b9510440B6ABB5CC02BB68ea1/',
+    description: 'Protocol Guild',
+    displayName: 'Protocol Guild',
```
Comment (lsankar4033) [inline packages/frontend/src/lib/sets.ts:159]: can you change displayName to include snapshot block for now? 

we can make it a mutating merkle tree later

File: packages/frontend/src/lib/sets.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ const SETS = [
   'medium-nft-trader',
 
   'nouns-forker',
+  'protocol-guild',
```
Comment (lsankar4033) [inline packages/frontend/src/lib/sets.ts:16]: I'd push the order up to the top of the list

**Review by lsankar4033 — CHANGES_REQUESTED** [review]

File: packages/frontend/src/lib/sets.ts (hunk 1 above)
Comment (DanTehrani) [reply packages/frontend/src/lib/sets.ts]: added!

File: packages/frontend/src/lib/sets.ts (hunk 2 above)
Comment (DanTehrani) [reply packages/frontend/src/lib/sets.ts]: moved to top

**Pull request merged**